zeebe/gateway-rest/pom.xml

```diff
@@ -254,6 +254,19 @@
       <groupId>com.jayway.jsonpath</groupId>
       <artifactId>json-path</artifactId>
       <scope>test</scope>
+      <exclusions>
+        <exclusion>
+          <groupId>net.minidev</groupId>
```

Comment:
@RomanJRW I have excluded the libraries that use the dependency 2.5.1 and when is necessary declared the 2.4.11
Please let me know if I should only ignore this case, so I can revert and use only the `parent/pom.xml`

To validate this, it can be used `mvn dependency:tree`